apps/dev/src/breadcrumbs/breadcrumbs-demo.component.html

```diff
@@ -1,89 +1,89 @@
 <div style="margin-top: 16px;">
   <dt-breadcrumbs>
-    <dt-breadcrumbs-item>dt-breadcrumbs-item component</dt-breadcrumbs-item>
+    <a DtBreadcrumbsItem2>dt-breadcrumbs-item component</a>
```

Comment:
The selector for the DtBreadcrumbsItem2 should stay at `'a[dt-breadcrumbs-item], a[dtBreadcrumbsItem]`
```suggestion
    <a DtBreadcrumbsItem>dt-breadcrumbs-item component</a>
```

This applies to the whole file.